src/ensembl/config.ts

```diff
@@ -1,6 +1,9 @@
 export default {
-  // version number
-  version: '0.2.0',
+  // Version numbers
+  app_version: '0.2.0',
+  privacy_policy_version: '2.0.0',
+  privacy_policy_cookie_name: 'ENSEMBL_PRIVACY_POLICY',
+  privacy_policy_cookie_expiry: 'Sun, 01 Jan 2040 00:00:00 GMT',
```

Comment:
These three extra fields seem to serve one single purpose of showing the privacy banner. I would put them in a constants file in the `shared/privacy-banner` folder, not in the general config.

Reply:
Another thing, which I talked about with @imran-ebi, is that we may want to change the storage mechanism for whether the user has seen and agreed to the banner from cookies to localStorage. LocalStorage is much more spacious than cookies (which are up to 4 kb or something?), and stays in the browser, whereas cookies are sent back and forth between the server and the browser. This doesn't look like something that belongs in cookies.

Reply:
Do you have an example of such usage that I can have a look?

Reply:
> privacy_policy_cookie_expiry: 'Sun, 01 Jan 2040 00:00:00 GMT',

This looks like "never" 😃 If we move privacy banner data to localStorage, we won't have to specify the expiry date.

Reply:
> Do you have an example of such usage that I can have a look?

@jyo search the project for browser-storage-service or species-selector-storage-service.